.gitignore

```diff
@@ -1,3 +1,3 @@
 /target/
 **/*.rs.bk
-Cargo.lock
+# Cargo.lock
```

Comment:
Typically libraries don't check in Cargo.lock.

Relevant advice from Rust docs:
"If you’re building a non-end product, such as a rust library that other rust packages will depend on, put Cargo.lock in your .gitignore"

https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html

Reply:
ah that's why, cool will update